llvm/lib/Transforms/InstCombine/InstCombineAtomicRMW.cpp

```diff
@@ -68,6 +68,12 @@ bool isSaturating(AtomicRMWInst& RMWI) {
     case AtomicRMWInst::FMin:
       // minnum(x, -inf) -> +inf
       return CF->isNegative() && CF->isInfinity();
+    case AtomicRMWInst::FMaximum:
```

Comment:
This is an extra optimization that should be done in a follow up, not the initial base patch

Reply:
Thanks, removed for now.